op.c

```diff
@@ -14839,6 +14852,12 @@ Perl_ck_subr(pTHX_ OP *o)
                 STRLEN len;
                 const char* str = SvPV(*const_class, len);
                 if (len) {
+                    if (!FEATURE_BAREWORD_FILEHANDLES_IS_ENABLED
+                        && !is_standard_filehandle_name(str)
+                        && (const_op->op_private & OPpCONST_BARE)) {
```

Comment:
Is there a possibility that `const_op` remains NULL by this point? Should there be a test on it? If not, should it be `assert()`ed?

Reply:
That's not possible, this code is conditional on `const_class` and `const_class` is set from the op used to set `const_op` at the same point `const_op` is set.

If this code wasn't also conditional on `SvPV(*const_class)` I would have made the if() conditional on `const_op` instead and moved initializing `const_class` within the if() block.

I'll add an assert().